File: eSalon/eSalon.Model/StavkeRezervacije.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eSalon.Model
{
    public class StavkeRezervacije
    {
        public int StavkeRezervacijeId { get; set; }

        public int UslugaId { get; set; }

        public int RezervacijaId { get; set; }

        public decimal? Cijena { get; set; }
        public  string ? UslugaNaziv {  get; set; }
        public int ?  Trajanje {  get; set; }

        public decimal? OriginalnaCijena { get; set; } 
        public bool ImaPopust => OriginalnaCijena > Cijena;

        public byte[]? Slika {  get; set; }

    }
}

File: eSalon/UI/esalon_mobile/lib/models/stavke_rezervacije.dart
import 'package:json_annotation/json_annotation.dart';

part 'stavke_rezervacije.g.dart';

@JsonSerializable()
class StavkeRezervacije {
  int? stavkeRezervacijeId;
  int? uslugaId;
  int? rezervacijaId;
  double? cijena;
  String? uslugaNaziv;
  int? trajanje;
  double? originalnaCijena;
  bool? imaPopust;
  String? slika; 

  StavkeRezervacije({
    this.stavkeRezervacijeId,
    this.uslugaId,
    this.rezervacijaId,
    this.cijena,
    this.uslugaNaziv,
    this.trajanje,
    this.originalnaCijena,
    this.imaPopust,
    this.slika,
  });

  factory StavkeRezervacije.fromJson(Map<String, dynamic> json) =>
      _$StavkeRezervacijeFromJson(json);

  Map<String, dynamic> toJson() => _$StavkeRezervacijeToJson(this);
}